File: testdata/repos/commons/templates/create_formula/languages/java11/helloWorld/src/main/java/com/ritchie/formula/Formula.java
package com.ritchie.formula;

import com.github.tomaslanger.chalk.Chalk;

public class Formula {

    private final String inputText;
    private final boolean inputBoolean;
    private final String inputList;
    private final String inputPassword;

    public Formula(String inputText, boolean inputBoolean, String inputList, String inputPassword) {
        this.inputText = inputText;
        this.inputBoolean = inputBoolean;
        this.inputList = inputList;
        this.inputPassword = inputPassword;
    }

    public void Run() {
        System.out.println("Hello World!");

        System.out.println(Chalk.on(String.format("My name is %s.", inputText)).green());

        if (inputBoolean) {
            System.out.println(Chalk.on("I've already created formulas using Ritchie.").cyan());
        } else {
            System.out.println(Chalk.on("I'm excited in creating new formulas using Ritchie.").red());
        }

        System.out.println(Chalk.on(String.format("Today, I want to automate %s.", inputList)).yellow());

        System.out.println(Chalk.on(String.format("My secret is %s.", inputPassword)).magenta());
    }
}

File: testdata/repos/commons/templates/create_formula/languages/kotlin-jdk8/helloWorld/src/main/kotlin/com/ritchie/formula/Formula.kt
package com.ritchie.formula
import com.github.tomaslanger.chalk.Chalk;

class Formula(private val inputText: String, private val inputBoolean: Boolean, private val inputList: String, private val inputPassword: String) {
    fun Run() {
        println("Hello World!")
        println(Chalk.on(String.format("My name is %s.", inputText)).green())
        if (inputBoolean) {
            println(Chalk.on("I've already created formulas using Ritchie.").cyan())
        } else {
            println(Chalk.on("I'm excited in creating new formulas using Ritchie.").red())
        }
        println(Chalk.on(String.format("Today, I want to automate %s.", inputList)).yellow())
       println(Chalk.on(String.format("My secret is %s.", inputPassword)).magenta())
    }
}
